src/SUMMARY.md

```diff
@@ -65,3 +65,11 @@
     - [A Simple Arithmetic Circuit](./halo2-for-dummies/plonkish/simple-arithmetic-circuit.md)
     - [Transforming to PLONKish Arithmetization](./halo2-for-dummies/plonkish/transforming-to-plonkish-arithmetization.md)
   - [A Simple Halo 2 Program](./halo2-for-dummies/simple-example/section.md)
+- [zkdatabase](./zkdatabase/chapter.md)
```

Comment:
Could you change the title's format

- zkdatabase -> zkDatabase
- b-tree -> B-Tree
- accumulation -> Accumulation
....

There are some issue with the sidebar:
`[ipfs] (./zkdatabase/distributed-storage-engine/ipfs.md)` there is an unnecessary extra space that break the MD link format.